src/core/lib/security/authorization/rbac_policy.h

```diff
@@ -162,15 +172,22 @@ struct Rbac {
   };
 
   Rbac() = default;
-  Rbac(Rbac::Action action, std::map<std::string, Policy> policies);
+  Rbac(Rbac::Action action, std::map<std::string, Policy> policies,
+       absl::string_view name);
```

Comment:
Please make `name` the first parameter, since it seems a little more logical that way.